go.mod

```diff
@@ -4,6 +4,9 @@ go 1.21.6
 
 replace github.com/yudai/gojsondiff v1.0.0 => github.com/Kong/gojsondiff v1.3.0
 
+// TODO: remove me
+replace github.com/kong/go-database-reconciler v1.10.0 => github.com/kong/go-database-reconciler v0.0.0-20240513201015-0cc539e9aad8
```

Comment:
Note: this PR is blocked until we get a new release of go-database-reconciler (preferably after [this PR](https://github.com/Kong/go-database-reconciler/pull/90) is merged).